src/naive/time/mod.rs

```diff
@@ -792,7 +792,8 @@ impl NaiveTime {
     }
 
     /// Returns a triple of the hour, minute and second numbers.
-    fn hms(&self) -> (u32, u32, u32) {
+    #[doc(hidden)]
+    pub fn hms(&self) -> (u32, u32, u32) {
```

Comment:
I'm open to making this a part of the public API, though I think we should probably yield `u8` instead of `u32` (even if this is different from other methods right now).